api/cluster.go

```diff
@@ -0,0 +1,29 @@
+package api
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/raintank/metrictank/api/middleware"
+	"github.com/raintank/metrictank/api/models"
+	"github.com/raintank/metrictank/api/rbody"
+	"github.com/raintank/metrictank/cluster"
+)
+
+func (s *Server) getNodeStatus(ctx *middleware.Context) {
+	rbody.WriteResponse(ctx, rbody.NewJsonResponse(200, cluster.ThisNode, ""))
```

Comment:
cluster.Node now implements json.Marshaler and its MarshalJSON method is thread safe.